client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -1,155 +1,102 @@
-
-	const { url: yoastSeoPageUrl } =
-		menuItems.find( ( { slug }: { slug: string } ) => slug === 'wpseo_dashboard' ) ?? {};
-
-	const translate = useTranslate();
-
-	useEffect( () => {
-		if ( ! postsPageUrl || ! yoastSeoPageUrl ) {
-			selectedSiteId && dispatch( requestAdminMenu( selectedSiteId ) );
+const ThankYouContainer = styled.div`
+	.marketplace-thank-you {
+		margin-top: var( --masterbar-height );
+		img {
+			height: auto;
 		}
-	}, [ dispatch, postsPageUrl, selectedSiteId, yoastSeoPageUrl ] );
+	}
+`;
 
-	useEffect( () => {
-		if (
-			isPluginInstalledDuringPurchase &&
-			transferStatus !== 'complete' &&
-			! isFetchingTransferStatus
-		) {
-			setTimeout( () => {
-				selectedSiteId && dispatch( fetchAutomatedTransferStatus( selectedSiteId ) );
-				setPollCount( ( c ) => c + 1 );
-			}, 1500 );
-		}
-	}, [
-		isFetchingTransferStatus,
-		isPluginInstalledDuringPurchase,
-		pollCount,
-		setPollCount,
-		dispatch,
-		transferStatus,
-		selectedSiteId,
-	] );
+const MarketplaceThankYou = ( { productSlug } ): JSX.Element => {
+	const translate = useTranslate();
+	const siteId = useSelector( getSelectedSiteId );
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const plugin = useSelector( ( state ) => getPluginOnSite( state, siteId, productSlug ) );
+	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
 
-	/* TODO: Make all these items product-dependent */
 	const thankYouImage = {
-		alt: 'yoast logo',
-		src: yoastInstalledImage,
+		alt: 'success image',
+		src: successImage,
 	};
 
-	const yoastSetupSection = {
-		sectionKey: 'yoast_whats_next',
+	const setupURL = plugin?.action_links?.Settings
+		? plugin.action_links.Settings
+		: `${ siteAdminUrl }plugins.php`;
+
+	const setupSection = {
+		sectionKey: 'setup_whats_next',
 		sectionTitle: translate( 'What’s next?' ),
 		nextSteps: [
 			{
-				stepKey: 'yoast_whats_next_plugin_setup',
+				stepKey: 'whats_next_plugin_setup',
 				stepTitle: translate( 'Plugin setup' ),
 				stepDescription: translate(
-					'Get to know Yoast SEO and customize it, so you can hit the ground running.'
+					'Get to know your plugin and customize it, so you can hit the ground running.'
 				),
 				stepCta: (
-					<FullWidthButton
-						href={ yoastSeoPageUrl }
-						primary
-						busy={ isRequestingMenu }
-						// TODO: Menu links are not properly loading on initial request, post transfer so yoastSeoPageUrl will remain empty post transfer
-						// This should be fixed with perhaps a work around to periodically poll for the menu with various domains until it loads
-						// or maybe blocking the user from entering this flow until a domain acquires SSL
-						disabled={ ! yoastSeoPageUrl }
-					>
-						{ translate( 'Get started' ) }
+					<FullWidthButton href={ setupURL } primary busy={ ! plugin }>
+						{ translate( 'Manage plugin' ) }
 					</FullWidthButton>
 				),
 			},
 			{
 				stepKey: 'yoast_whats_next_view_posts',
```

Comment:
I think this is now a generic component, so we probably should get rid of the `yoast_` prefix:

```suggestion
				stepKey: 'whats_next_view_posts',
```

Reply:
Good catch!